H/HiGHS/build_tarballs.jl

```diff
@@ -72,5 +72,6 @@ build_tarballs(
     platforms,
     products,
     dependencies;
-    preferred_gcc_version = v"4.9",
+    preferred_gcc_version = v"5",
```

Comment:
You may want to go to 6
```suggestion
    preferred_gcc_version = v"6",
```
We have 4.8, 5, etc..., so between 4.9 and 5 it doesn't change much :wink: